src/operators/validator.operators.ts

```diff
@@ -1,26 +1,66 @@
-import 'reflect-metadata';
 import { Type } from '@nestjs/common';
+import { IsOptional, getMetadataStorage } from 'class-validator';
+
+/* Return the decorator properties of the given class.
```

Comment:
I'm mirroring the structure used in for the transformer omit/pick operators by writing a function to extract the field names registered with `class-validator`.